src/vm/man/vmadm.1m.md

```diff
@@ -390,6 +390,15 @@ tab-complete UUIDs rather than having to type them out for every command.
         JSON object on stdin (though it will refuse to work if stdin is a tty),
         or pass property=value arguments on the command line.
 
+	NOTE that property=value arguments may behave differently than JSON
```

Comment:
This uses tabs whereas the rest is space indented.

Pending the other thing about passing json directly on the cmdline, I'd reword this as something like

```
Many properties can be cleared by specifying their value as null in the JSON, e.g.

  { ... "zfs_snapshot_limit": null }

However this doesn't work via a direct `vmadm update prop=null` command